starter-core/src/main/java/io/micronaut/starter/feature/build/maven/templates/pom.rocker.raw

```diff
@@ -510,10 +510,17 @@ List<Property> properties
 @dependency.template("org.junit.jupiter", "junit-jupiter-api", "test", null)
 @dependency.template("org.junit.jupiter", "junit-jupiter-engine", "test", null)
 @dependency.template("io.micronaut.test", "micronaut-test-junit5", "test", null)
-} else if (features.testFramework().isKotlinTest()) {
-@dependency.template("io.micronaut.test", "micronaut-test-kotlintest", "test", null)
+
+} else if (features.testFramework().isKotlinTest() || features.testFramework().isKoTest()) {
```

Comment:
This could be simpler if this PR was merged: https://github.com/micronaut-projects/micronaut-starter/pull/411